### DISCUSSION TITLE
How to change the menu key?

### DISCUSSION THREAD

**Discussion by AtlaszFox** [Flazhik/Jukebox#2] [Q&A] [answered]
I cannot use the F4 key to open the menu. Is there a way that I can change it through the files?

**Comment by Flazhik** [comment, marked as answer]
Hey! Open **ULTRAKILL\Preferences\LocalPrefs.json**, create a new **cyberGrind.musicExplorer.keyBinding.CGMEMenu** entry and make it 286 for F5, 287 for F6 etc. Just make sure not to break json format.

**Reply by Carl03os** [reply]
How do you do correctly? I try it and it didn't work. can you give more information of to do it?

**Reply by Flazhik** [reply]
Which version do you have installed? All the versions below 1.6.1 aren't supported after the Violence layer update.
In case you have 1.6.1, did you install the mod manually or using r2modman?

**Reply by Carl03os** [reply]
I assume somewhere at and above 1.6.1 because the mod does work it did play custom music I put in the cybergrind file and I did it manually is that i don’t know how to put the entry in the json file

**Reply by Carl03os** [reply]
And before you ask I try put the entry to bind with F5 key but it didn’t work

**Reply by Flazhik** [reply]
The custom music itself is a part of the official functionality now, so it doesn't indicate that the mod was installed and is working. Please download the latest release and try to reinstall the mod.
If it works (extra buttons appear in Music section of the blue terminal), but bindings still don't work, just share json files and we'll fix things up.
